https://issues.redhat.com/browse/ACM-11181 Partial matching search (#6390)
open-cluster-management/rhacm-docs: observability/search_intro.adoc

  operator: Exists
----

[#search-operations]
== Search operations and data types

> You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:

Reviewer: What is the benefit or purpose of users searching with conditions? See my suggested rewrite, and make changes if I am misunderstanding the benefit. 

```suggestion
To identify what operations or data types you need to implement or modify, use conditions to find these operations or data types. You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:
```

> You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:

Author: @jc-berger thanks Jake, how about this? 
```suggestion
Specify your search query by using search operations as conditions. Characters such as `+>, >=, <, <=, !=+` are supported. See the following search operation table:
```

.Search operation table
|===

> | Operation | Data type | Description

Author: ```suggestion
| Default operation | Data type | Description
```

| `=`
| string, number
| This is the default operation.

| `!` or `!=`
| string, number
| This represents the _NOT_ operation, which means to exclude from the search results.

> | `<`, `<=`, `>`, `>=`

Reviewer 2: Not sure why the <= shows up weird in the rich preview. Everything else, looks good.

| number
| 

> | `TODO`

Reviewer 2: For dates, the default operation is ">".

> | `TODO`

Author: ```suggestion
| `>`
```

> | `<`, `<=`, `>`, `>=`

Author: ```suggestion
| `<`, `\<=`, `>`, `>=`
```

> | `<`, `\<=`, `>`, `>=`

Author: ```suggestion
| `<, <=, >, >=`
```

| date
| Dates matching the last hour, day, week, month, and year.

> | `*` 

Author: ```suggestion
| `\*` 
```

> | `\*` 

Author: ```suggestion
| \`*` 
```

> | \`*` 

Author: ```suggestion
| `*`\ 
```

> | `*`\ 

Author: ```suggestion
| `*`
```
